catalog/spatial/geocoding/spatial-geocoding-lucene/src/test/java/org/codice/ddf/spatial/geocoding/query/TestGeoNamesQueryLuceneIndex.java

```diff
@@ -236,27 +239,27 @@ public void testQueryWithNoResults() throws IOException, ParseException {
     }
 
     @Test(expected = IllegalArgumentException.class)
-    public void testBlankQuery() {
+    public void testBlankQuery() throws GeoEntryQueryException {
         directoryIndex.query("", 1);
     }
 
     @Test(expected = IllegalArgumentException.class)
-    public void testNullQuery() {
+    public void testNullQuery() throws GeoEntryQueryException {
         directoryIndex.query(null, 1);
     }
 
     @Test(expected = IllegalArgumentException.class)
-    public void testQueryZeroMaxResults() {
+    public void testQueryZeroMaxResults() throws GeoEntryQueryException {
         directoryIndex.query("phoenix", 0);
     }
 
     @Test(expected = IllegalArgumentException.class)
-    public void testQueryNegativeMaxResults() {
+    public void testQueryNegativeMaxResults() throws GeoEntryQueryException {
         directoryIndex.query("phoenix", -1);
     }
 
     @Test(expected = GeoEntryQueryException.class)
-    public void testQueryNoExistingIndex() throws IOException {
+    public void testQueryNoExistingIndex() throws IOException, GeoEntryQueryException {
         doReturn(false).when(directoryIndex)
                 .indexExists(directory);
         directoryIndex.query("phoenix", 1);
```

Comment:
Just a nit, but some of these strings are used multiple times, and can be static final constants instead